src/gui/MainWindow.cpp

```diff
@@ -407,13 +411,44 @@ MainWindow::MainWindow()
         m_ui->globalMessageWidget->showMessage(
             tr("Access error for config file %1").arg(config()->getFileName()), MessageWidget::Error);
     }
-
+#ifdef WITH_XC_HTTP
+    if (config()->get("Http/Enabled", false).toBool() && !config()->get("Http/DeprecationNoticeShown", false).toBool()) {
+        // show message after tab widget dismissed all messages
+        connect(m_ui->tabWidget, SIGNAL(messageDismissGlobal()), this, SLOT(showKeePassHTTPDeprecationNotice()));
+    }
+#endif
 }
 
 MainWindow::~MainWindow()
 {
 }
 
+void MainWindow::showKeePassHTTPDeprecationNotice()
+{
+    displayGlobalMessage(tr("<p>It looks like you are using KeePassHTTP for browser integration.<br>"
+                                "This feature has been deprecated and will be removed in the future.<br>"
+                                "Please switch to keepassxc-browser instead! For help with migration,<br>"
+                                "visit our <a href=\"https://keepassxc.org/docs/keepassxc-browser-migration\">"
+                                "keepassxc-browser migration guide</a>.</p>"),
```

Comment:
Remove most of the breaks in this message, it does not render well on Windows.

Reply:
Same problem as above. I had it with only one break and it looked far better on Windows. But other systems, other font sizes. The text will increase the minimum window width quite significantly. On small screens it may overflow. I tried to enable word wrapping, but that kind of caused the widget layout to fall apart.